src/Transports/MassTransit.SqlTransport.PostgreSql/SqlTransport/PostgreSql/PostgresDatabaseMigrator.cs

```diff
@@ -1146,7 +1149,7 @@ await connection.Connection.ExecuteScalarAsync<int>(string.Format(GrantRoleSql,
             result = await connection.Connection.ExecuteScalarAsync<int>(string.Format(RoleExistsSql, options.Username)).ConfigureAwait(false);
             if (result != 1)
             {
-                await connection.Connection.ExecuteScalarAsync<int>(string.Format(CreateUserSql, options.Role, options.Username, options.Password))
+                await connection.Connection.ExecuteScalarAsync<int>(string.Format(CreateUserSql, options.Role, options.Username, SqlTransportPasswordRetriever.GetPassword(options)))
```

Comment:
Pretty sure storing an access token in the RDS database isn't the right way to handle it, as that token would change over time.

Reply:
@phatboyg that is correct - that we don't quite store the access token that is retrieved from RDS.. In the current implementation, everytime `SqlTransportPasswordRetriever.GetPassword` is called a new token is generated - given it calls `RDSGenerateAuthToken` everytime the `Generate` Token method is called.

Reply:
So how would creating a user in the database with a tokenId even work then?

Reply:
@phatboyg  I reckon I misunderstood the earlier question..  I reckon the `connection` itself would require the password to be rotated with `connection.UsePeriodicPasswordProvider` and the statement side of it in `string.format` would have to be changed to a static password string given it essentially just creates a user..